src/lib.rs

```diff
@@ -625,9 +625,13 @@ impl LightningNode {
 
     /// Calculate the actual LSP fee for the given amount of an incoming payment.
     /// If the already existing inbound capacity is enough, no new channel is required.
+    /// The LSP may offer multiple fee rates, tied to different expiration dates.
+    /// Increased expiry dates mean higher fee rates.
+    /// This method returns the best offer within the given expiry.
     ///
     /// Parameters:
     /// * `amount_sat` - amount in sats to compute LSP fee for
+    /// * `expiry` - expiry time in seconds
```

Comment:
The docs here are not up-to-date. We probably don't want any changes

Reply:
Removed: 2a522c732a12c16c00c216b17e7199e9199a1819